rotkehlchen/tests/unit/test_data_updates.py

```diff
@@ -121,6 +123,42 @@
     },
 }
 
+ACCOUNTING_RULES_DATA = {
+    'accounting_rules': [
+        {
+            'event_type': HistoryEventType.SPEND.serialize(),
+            'event_subtype': HistoryEventSubType.FEE.serialize(),
+            'counterparty': 'test_counterparty',
+            'taxable': True,
+            'count_entire_amount_spend': True,
+            'count_cost_basis_pnl': True,
+            'method': 'spend',
+            'accounting_treatment': None,
+            'links': {'count_cost_basis_pnl': 'include_crypto2crypto'},
+        },
+        {
```

Comment:
please do this over this file and other places you do this. Saves space

```suggestion
        }, {
```